content/en/docs/tutorials/security/seccomp.md

```diff
@@ -481,7 +480,7 @@ kubectl delete service fine-pod --wait
 kubectl delete pod fine-pod --wait --now
 ```
 
-## Create Pod that uses the container runtime default seccomp profile
+## Create Pod that uses the default seccomp profile for container runtime
```

Comment:
I think the change makes it harder to understand. The "default profile" belongs to the container runtime.
The Seccomp Profile is not for the container runtime, but is rather used by it to restrain the container process.

```suggestion
## Create Pod that uses the container runtime default seccomp profile
```